doc/en/download/Best Practice.rst

```diff
@@ -6,12 +6,30 @@ Best Practice
 .. _example_common_utilities:
 
 Helper Utilities
-----------------
+================
 
 Testplan provides helper functions and a predefined testsuite that make it
 easy for the user to add common testplan execution infomation - such as
 env var, pwd, log file, driver metadata - to the test report.
 
+DriverLogCollector
+------------------
+You can specify your custom log collector using :py:class:`~testplan.common.utils.helper.DriverLogCollector`.
+It will attach the specified log files to the report for each driver.
+Example:
+    .. code-block:: python
+
+      from testplan.common.utils import helper
```

Comment:
Add one or two empty line to be inline with how we format code blocks of similar nature.